libraries/package-extractor/src/PackageExtractor.ts

```diff
@@ -415,25 +425,31 @@ export class PackageExtractor {
       const additionalFolderExtractorOptions: IExtractorOptions = {
         ...options,
         sourceRootFolder: additionalFolderPath,
-        targetRootFolder: options.targetRootFolder
+        targetRootFolder
       };
       await this._extractFolderAsync(additionalFolderPath, additionalFolderExtractorOptions, state);
     }
 
     switch (linkCreation) {
       case 'script': {
-        const sourceFilePath: string = path.join(scriptsFolderPath, createLinksScriptFilename);
-        if (!options.createArchiveOnly) {
-          terminal.writeLine(`Creating ${createLinksScriptFilename}`);
-          await FileSystem.copyFileAsync({
-            sourcePath: sourceFilePath,
-            destinationPath: path.join(targetRootFolder, createLinksScriptFilename),
-            alreadyExistsBehavior: AlreadyExistsBehavior.Error
```

Comment:
This is changing the already-exists behavior.

Reply:
Yeah, that should be fine. I'm not concerned about overwriting this, since we already do a confirmation that the extraction target folder doesn't exist at the beginning of execution